GCHP 14.5.3 release

yantosca opened the pull request:
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Describe the update
This PR seeks to merge the `release/14.5.3` branch into `main` in anticipation of the GCHP 14.5.3 release. 

The `release/14.5.3` branch contains
- Commits from the `dev/no-diff-to-benchmark` branch
- ReadTheDoc updates for GCClassic 14.5.3
- Updated version numbers to 14.5.3 in the usual places.

### Expected changes
This is a no-diff-to-benchmark update

Tagging @msulprizio @lizziel

yantosca committed:
GEOS-Chem update: Merge PR #2713 (CESM reads AEIC2019 every timestep)

This commit informs the GCHP superproject about the following
commits that were pushed to the GitHub geoschem/geos-chem repository:

37da3740b Merge PR #2713 (Read AEIC2019 every timestep in CESM)

This PR updates the HEMCO configuration file used with CESM to
read AEIC emissions every model timestep. This avoids introducing
small differences when the model runs are broken up in time.

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>

yantosca committed:
RTD updates: AWS-CLI documentation

docs/source/geos-chem-shared-docs
- Submodule update to c769e00

CHANGELOG.md
- Updated accordingly

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>

lizziel committed:
Update HEMCO submodule with PRs #299, #304, and #306

Submodule src/GCHP_GridComp/GEOSChem_GridComp/HEMCO/HEMCO 0ae25d2..d7563e7:
  > Merge pull request #306 from geoschem/bugfix/remove_cesm_core_dependency
  > Merge pull request #299 from K20shores/ci
  > Merge PR #304 (Add .zenodo.json; Update jinja2 to 3.1.5)

yantosca committed:
Merge branch 'dev/no-diff-to-benchmark' into release/14.5.3

This merge brings updates from the "no-diff-to-benchmark" development
stream into the release/14.5.3 branch, in anticipation of the
GCHP 14.5.3 release.

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>

yantosca committed:
Merge docs/dev into release/14.5.3

This merge brings ReadTheDocs documentation updates for GCHP 14.5.3
(in the docs/dev branch) into the release/14.5.3 branch.

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>

yantosca committed:
Update version numbers and submodule hashes for 14.5.3 release

Updated version numbers to 14.5.3 in:
- CHANGELOG.md
- CMakeLists.txt
- docs/source/conf.py

Updated src/GEOS-Chem submodule to the 14.5.3 release commit
Updated src/HEMCO submodule to the 3.10.2 release commit

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>

lizziel commented on CHANGELOG.md:
```diff
@@ -9,6 +9,10 @@ This file documents all notable changes to the GCHP wrapper repository starting
 
+## [14.5.3] - 2025-03-05
+### Added
+- Updated RTD docs to state that users wishing not to open AWS account can use AWS CLI commands with `--no-sign-request`
```
This needs an entry for submodule version updates of HEMCO and GEOS-Chem.

yantosca replied:
Thanks for catching that @lizziel.  Now fixed in b13dadc.

yantosca committed:
GCHP 14.5.3 release

This is the official release commit of GCHP 14.5.3.

Updated version numbers in:
- CHANGELOG.md
- CMakeLists.txt
- docs/source/conf.py

Updated GEOS-Chem submodule to version 14.5.3
Updated HEMCO submodule to version 3.10.2

Updated geos-chem-shared-docs submodule to commit c769e00

Signed-off-by: Bob Yantosca <yantosca@seas.harvard.edu>
